frontend/src/modules/app/SidebarClickable.tsx

```diff
@@ -62,7 +68,11 @@ const useStyles = makeStyles((theme: Theme) => ({
 /**
  * SidebarClickable
  *
- * The main FullPageNavigation panel of the application.
+ * @component The main FullPageNavigation panel of the application.
+ *
+ * @param {function} toggleStatus - Callback with toggleStatus
```

Comment:
This needs to describe what the `toggleStatus` callback is supposed to do. In general, when a callback is provided as a parameter, the parameter description should describe what the callback does. For example, is `toggleStatus` supposed to set the value of a boolean `useState` created elsewhere? Or is it supposed to do something else?

Reply:
Yes this will show/hide sidebar drawer. I am updating it as following
```
 * @param {function} toggleStatus - Callback with toggleStatus to show/hide Sidebar drawer status
 ```